src/docs/firmware/ti50-ap-ro.md

```diff
@@ -0,0 +1,51 @@
+---
+prev: write-protect
+next: false
+---
+# Disabling RO firmware protection on Ti50
+
+::: tip
+This is only applicable to devices with a Ti50 security chip, which is typically found in devices released during or after mid-2023.
+:::
+
+<br>
+
+## Step 1: Verify the device has a Ti50
+
+1. Open VT-2 (`ctrl` + `alt` + `f2` (right arrow)).
+2. Login as `root`.
+3. Run `gsctool -a -I | grep AllowUnverifiedRo`. 
+4. If that command has any output, your device has a Ti50. If you get no output or an error, your device either has a Cr50 or no GSC at all.
+
+## Step 2: Disable RO verification.
+1. Unlock the GSC by running `gsctool -a -o`. TODO: explain the unlocking process better (make a GSC unlocking page)
```

Comment:
Pending ToDo

Reply:
Whoops, missed that. Looking back at this, I think that change is outside of the scope of the PR. I can open an issue to track this if you want.

Reply:
> (...) I can open an issue to track this if you want.
Do that